packages/embark/src/cmd/reset.js

```diff
@@ -17,16 +17,19 @@ exports.paths = new Set([
   'node_modules/.cache'
 ]);
 
-exports.reset = async ({
-  doneMessage = green('Done!'),
-  removePaths = exports.paths
-} = {}) => {
+async function reset() {
   const completePaths = [...removePaths].map(relative => join(dappPath, relative));
 
   console.log(green('Removing the following files:\n'));
   completePaths.forEach(path => console.log(`${path}`));
   await Promise.all(
     completePaths.map(path => rimraf(path))
   );
+  let doneMessage = green('Done!');
```

Comment:
I'd rather see these expressed like this:

```js
console.log(chalk`{green Done!}`);
```;

Reply:
Agreed, that's better. I didn't know about chalk's tagged template literals at the time, so that was my way of getting away from the `colors` package and its global prototype patching.